sched/queue/memory/simple.go

```diff
@@ -0,0 +1,81 @@
+package memory
+
+// In-memory Scheduler Queue
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/sched"
+	"github.com/scootdev/scoot/sched/queue"
+	"time"
+)
+
+func NewSimpleQueue() (queue.Queue, chan sched.Job) {
```

Comment:
Nit: using this to demo, maybe allowing the caller to specify the queue capacity would be nice.